### ISSUE TITLE
Geth mode: Contract interface does not appear either in Mist wallet nor Mist in Parity home screen.

### ISSUE THREAD

**Issue by ferni** [ethcore/parity#1913]
The button "Show contract info" that's in Mist's contract screen, which opens the options for interacting with the contract, does not appear. I've tried running "parity --geth", using the Ethereum Wallet software (Ubuntu), as well as using Parity's home screen to access the Ethereum Wallet from the browser. In both cases the section for interacting with a watched contract does not appear. I tried this when attempting to split ETC funds , using the splitter contract. I've tried using this contract both in the ETH chain as well as the ETC chain.

**Comment by tomusdrw** [comment]
This button is displayed only if contract has some code. Can you check the result of:

```bash
$ curl -H "Content-Type: application/json" -X POST --data '{"jsonrpc":"2.0","method":"eth_getCode","params":["0x<address>", "latest"],"id":1}' localhost:8545
```

(also make sure you are connected to the right network (homestead/morden)

**Comment by ferni** [comment]
fer@fer-pc-ubuntu:~$ curl -H "Content-Type: application/json" -X POST --data '{"jsonrpc":"2.0","method":"eth_getCode","params":["0xAA1A6e3e6EF20068f7F8d8C835d2D22fd5116444", "latest"],"id":1}' localhost:8545
{"jsonrpc":"2.0","result":"0x6060604052361561001f5760e060020a60003504630f2c93298114610028575b6100005b610002565b6100c1600435602435600080547f16c72721000000000000000000000000000000000000000000000000000000006060908152600160a060020a03909116906316c727219060649060209060048187876161da5a03f11561000257505060405151905080156100b25750604051600160a060020a038416908290349082818181858883f193505050505b156100d3575060015b92915050565b60408051918252519081900360200190f35b600060009054906101000a9004600160a060020a0316600160a060020a03166316c727216040518160e060020a0281526004018090506020604051808303816000876161da5a03f1156100025750506040515115905080156101505750604051600160a060020a038316908290349082818181858883f193505050505b15610023575060016100bb56","id":1}fer@fer-pc-ubuntu:~$

**Comment by ferni** [comment]
![image](https://cloud.githubusercontent.com/assets/653586/17641757/c1957a32-6103-11e6-867c-50173c97ec4a.png)

Screenshot from Ethereum Wallet

**Comment by tomusdrw** [comment]
It's probably some bug in Ethereum Wallet, have you tried if it works correctly with Geth?

**Comment by ferni** [comment]
Yes, it works correctly with Geth.

**Comment by gavofyork** [comment]
if this turns out to be a bug in parity, i'm happy to fix.

however, without further debugging, we can only conclude that it is some bug in the ethereum wallet. reporting it to them might help in getting it properly diagnosed.

**Comment by ferni** [comment]
It seems it was indeed a bug in Ethereum Wallet, https://github.com/ethereum/meteor-dapp-wallet/issues/279 

The issue appears to be fixed in 0.8.2
